src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius/sylius_mailer.yml

```diff
@@ -25,9 +25,13 @@ sylius_mailer:
         password_reset:
             subject: sylius.emails.user.password_reset.subject
             template: "@SyliusCore/Email/passwordReset.html.twig"
+        #deprecated since Sylius 1.13 and will be removed in 2.0, use verification_token instead
         account_verification_token:
             subject: sylius.emails.user.verification_token.subject
             template: "@SyliusCore/Email/accountVerification.html.twig"
+        verification_token:
+            subject: sylius.emails.user.verification_token.subject
+            template: "@SyliusCore/Email/verification.html.twig"    
```

Comment:
I'd expect the template to be named the same as the configured email since that's the case in all other emails (excluding resending from this, since we reuse the same template)